librosa/core/constantq.py

```diff
@@ -259,15 +246,10 @@ def cqt(y, sr=22050, hop_length=512, fmin=None, n_bins=84,
                                                res_type=res_type,
                                                scale=True)
             my_sr /= 2.0
-            assert my_hop % 2 == 0
             my_hop //= 2
 
         # Compute a dynamic hop based on n_fft
```

Comment:
And this comment is now wrong.  If you don't think __fft_response is self-explanatory, you should give the function a better name.  No need to comment.